### PR TITLE
Add more parameters to the tuning playbook

### PR DESCRIPTION
**Pull request by k32**

### CODE REVIEW

File: templates/cluster-override.conf [hunk 1]
```diff
@@ -3,12 +3,16 @@
...
   role = "core"
+  tlog_push_mode = "{{ push_mode | default('async') }}"
+  rpc_module = "{{ rpc_module | default('gen_rpc') }}"
 }
 {% elif emqx_db_role == "replicant" %}
 db {
   backend = "rlog"
   role = "replicant"
-  core_nodes = "{{ cores }}"
+  core_nodes = "emqx@{{ cores | join(', emqx@' )}}"
```
Comment (k32) [inline templates/cluster-override.conf:13]: It's a horrible hack

Comment (thalesmg) [reply templates/cluster-override.conf:13]: Maybe we can define another var in a similar way to what is done for `cores`?

https://github.com/k32/emqx-ansible-cdk-helpers/blob/409d73acceaddca8e78d7a56b026655909534706/tuning.yml#L80

Comment (thalesmg) [reply templates/cluster-override.conf:13]: Actually: is this change needed? It has been working using only `cores` so far. :thinking:

**Comment by thalesmg** [comment]
I made a small spinoff in #4 because I changed a few details.

**Comment by k32** [comment]
@thalesmg Please go ahead with your spinoff

**Pull request closed without merging**